Extractor/RebrowseTriggerManager.cs

```diff
@@ -186,26 +187,23 @@ CancellationToken token
                 .Select(
                     (dv, i) =>
                         (
-                            nodes[i].Item1,
+                            _uaClient.GetUniqueId(nodes[i].Item1),
                             dv.GetValue<DateTime>(default).ToUnixTimeMilliseconds()
                         )
                 )
                 .ToDictionary(item => item.Item1, item => item.Item2);
```

Comment:
Use `ToDictionary` directly instead of going through a tuple.

Reply:
This may be necessary because there's no way to get the index in `ToDicationary`.

Reply:
Could you use `Zip`?

Reply:
With `Zip` we'd have to create anonymous objects that can be used in `ToDictionary`.